netlify.conf

```diff
@@ -0,0 +1,11 @@
+[Build & deploy]
+
+[Continuous Deployment]
+
+[Build settings]
+Repository = github.com/kubevirt/user-guide
+Base directory = Not set
+Build command = source /opt/buildhome/python3.7/bin/activate; pip3 install mkdocs mkdocs-awesome-pages-plugin; sed -i 's/site_url: https:\/\/kubevirt.io\/docs/site_url: https:\/\/kubevirt.io\//' /opt/build/repo/mkdocs.yml; sed -i 's/docs_dir: docs/docs_dir:/' /opt/build/repo/mkdocs.yml; echo "*** BEGIN /opt/build/repo/mkdocs.yml ***"; cat /opt/build/repo/mkdocs.yml; echo "*** END /opt/build/repo/mkdocs.yml ***"; mkdocs build -f /opt/build/repo/mkdocs.yml -d /opt/build/repo/site
```

Comment:
Isn't this redundant with `netlify.toml`?

Reply:
it is.  I will remove.